packages/react-components/package.json

```diff
@@ -22,13 +22,10 @@
     "start": "yarn storybook",
     "docs": "api-extractor run --config=config/api-extractor.local.json --local",
     "build:local": "tsc -p ./tsconfig.lib.json --module esnext --emitDeclarationOnly && node ../../scripts/typescript/normalize-import --output ./dist/packages/react-components/src && yarn docs",
-    "prestorybook": "yarn prepare-storybook-local-build",
-    "storybook": "start-storybook --port 3000 -s ./public --no-manager-cache",
-    "prestorybook:docs": "yarn prepare-storybook-local-build",
-    "storybook:docs": "start-storybook --port 3000 -s ./public --docs --no-manager-cache",
```

Comment:
Intended changes? Just to confirm if `prepare-storybook-local-build` is what this PR is addressing and if the docs stuff is intentional too.

Reply:
yes, this was quick fix added in https://github.com/microsoft/fluentui/pull/21247

- this replaces it with unified solution across all packages
- storybook:docs just calls local `storybook` script to remove repetition (both are called with same arguments   except `--docs`